src/__tests__/resources/revisions3D.integration.spec.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright 2019 Cognite AS
+
+import MockAdapter from 'axios-mock-adapter';
+import { readFileSync } from 'fs';
+import { API } from '../../resources/api';
+import {
+  CreateRevision3D,
+  Model3D,
+  Revision3D,
+  Tuple3,
+  UpdateRevision3D,
+  UploadFileMetadataResponse,
+} from '../../types/types';
+import { setupClient, simpleCompare, retryInSeconds } from '../testUtils';
+
+describe('Revision3d integration test', async () => {
+  let client: API;
+  let mock: MockAdapter;
+
+  beforeAll(async () => {
+    jest.setTimeout(30000);
+    client = await setupClient();
+  });
+
+  const now = Date.now();
+  let revisions: Revision3D[];
+  let file: UploadFileMetadataResponse;
+  let model: Model3D;
+
+  test('create model', async () => {
+    const modelToCreate = { name: `Model revision test ${now}` };
+    model = (await client.models3D.create([modelToCreate]))[0];
+    expect(model).toBeTruthy();
+  });
+
+  test('create file', async () => {
```

Comment:
This is identical to the test in AssetMappings. Could we refactor it into a common function?

Reply:
How about a rule of two, if I copy it in 3rd place I will refactor it.
don't you agree?

Reply:
We can go for that rule when the # lines is low, but if you have 10 lines duplicated two places then I think it is worth to refactor. But lets skip this test then